test/e2e/operator/garden/internal/rotation/certificate_authorities.go

```diff
@@ -106,7 +106,8 @@ func (v *CAVerifier) AfterPrepared(ctx context.Context) {
 func (v *CAVerifier) ExpectCompletingStatus(g Gomega) {
 	g.Expect(helper.GetCARotationPhase(v.Garden.Status.Credentials)).To(Equal(gardencorev1beta1.RotationCompleting))
 	Expect(v.Garden.Status.Credentials.Rotation.CertificateAuthorities.LastCompletionTriggeredTime).NotTo(BeNil())
-	Expect(v.Garden.Status.Credentials.Rotation.CertificateAuthorities.LastCompletionTriggeredTime.After(v.Garden.Status.Credentials.Rotation.CertificateAuthorities.LastInitiationFinishedTime.Time)).To(BeTrue())
+	Expect(v.Garden.Status.Credentials.Rotation.CertificateAuthorities.LastCompletionTriggeredTime.Time.Equal(v.Garden.Status.Credentials.Rotation.CertificateAuthorities.LastInitiationFinishedTime.Time) ||
```

Comment:
Can you adapt similar occurrences in https://github.com/gardener/gardener/blob/ebca5ce49bd5ac6c3edd0bbbda76be44a963798a/test/e2e/gardener/shoot/internal/rotation as well, please? I think there are at least 3 of them (CA, ETCD, SA)